backend/src/main/java/gov/cdc/usds/simplereport/db/model/Person.java

```diff
@@ -260,4 +260,16 @@ public String getCounty() {
   public PersonRole getRole() {
     return role;
   }
+
```

Comment:
Specifications access fields using name strings. Put these strings with the class so they are all in one place. Put them into a sub class for scoping. Is this the best approach?

Reply:
The "standard" approach for this is to have a MetaModel object or class that contains this information—preferably auto-generated, but we don't have code generation and I kind of like that. So, in short, this is pretty much what you do, yeah. 😁 

Potentially worthwhile: moving this inner class to the class where specifications are built, instead of the model—honestly though until we have at least two or three of them, it doesn't matter a ton where they live.